CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # TRILLIAN Changelog
 
+## v1.0.9 - CloudSpanner Improvements & Internal API changes etc.
+
+Published 2018-04-17 07:01:46 +0000 UTC
+
+Changes are in progress (e.g. see PR #1037) to rework the internal signed root format used by the log RPC server to be more useful / interoperable. Currently they are mostly internal API changes to the log and map servers.

```

Comment:
I'm not sure the changes are just "internal API changes" -- running `git diff v1.0.8.. *proto` gives a lot of (external) API changes, some of which are not obviously back-compatible.  Can we summarize the API changes and think about whether they justify a higher-level bump in version?

Reply:
Apart from the signature changes most of the others seem to be Maps related or part of implementing preordered logs.

Reply:
The `signature` and `log_id` fields in `SignedLogRoot` have been deleted.  If any client code that uses the Trillian gRPC API references those fields, it will fail to compile with this update.  From a [semver](https://semver.org) perspective, that would imply that this should be v2.0.0.

Reply:
Oh, and adding the SLR to `Get*` responses is adding new functionality in back-compatible way, so would imply v1.1.0...

Reply:
I'll close this and create a new branch for 1.1.0 then.